src/instructlab/model/chat.py

```diff
@@ -789,6 +536,157 @@ def start_prompt(
         self._update_conversation(response_content.plain, "assistant")
 
 
+def chat_model(
+    ctx,
+    question,
+    model,
+    context,
+    session,
+    quick_question,
+    max_tokens,
+    endpoint_url,
+    api_key,
+    tls_insecure,
+    tls_client_cert,
+    tls_client_key,
+    tls_client_passwd,
+    model_family,
+    serving_log_file,
+    temperature,
+):
+    """Runs a chat using the modified model"""
+    # pylint: disable=import-outside-toplevel
+    # First Party
+    from instructlab.model.backends.common import is_temp_server_running
+
+    max_ctx_size = None
+    backend_type = ctx.obj.config.serve.server.backend_type
+    serve_server = ctx.obj.config.serve.server
```

Comment:
should be passed in

Reply:
this might need to be split up more even for future uses (non-primitives can't be pickled I don't think)